### DISCUSSION TITLE
Update Framework for Test Project

### DISCUSSION THREAD

**Discussion by pvoelker** [IntergatedCircuits/HidSharp#26] [Ideas]
Any objections if I put a pull request out there to update the test project framework?  Visual Studio is a bit cranky about .NET Core 2.0.

I can bump up to .NET Core 3.1 or take it all the way up to .NET 7 if we want to...  Thoughts?

I'm looking to use this to build a standard interface library for some low cost quiz box hardware....

**Comment by benedekkupper** [comment]
sure go ahead :) can you send me two commits, one bumping to the lowest still useful version, the second to the most current one?
